setup-environment

```diff
@@ -38,7 +38,7 @@ else
     fi
 
     for i in $(seq $#); do
-        setup_flex_arg="$(eval printf "%s" "\$$i")"
+        setup_flex_arg="$(eval printf "%s" "\${$i}")"
         case "$setup_flex_arg" in
             -b)
                 BUILDDIR="$(eval printf "%s" "\$$((i + 1))")"
```

Comment:
Hmmmm don't we need the above magic here as well? Would we be better off implementing actual arg parsing?